theme/templates/base.html

```diff
@@ -40,7 +40,12 @@
 <script type="text/javascript" src="{% static 'js/bootstrap.3.0.2.min.js' %}"></script>
 <script type="text/javascript" src="{% static 'js/bootstrap-extras.js' %}"></script>
 
-{% block extra_js %}{% endblock %}
+{% block extra_js %}
+<script>
+    const ROUTE_NAME = "{{ request.resolver_match.url_name }}";
```

Comment:
What is returned by `url_name` if the route is unnamed? We need to make sure that this constant resolves to an empty string in that case.

Reply:
"None" is returned for unnamed routes

Reply:
For unnamed routes the constant would be assigned the string `None`.
![image](https://user-images.githubusercontent.com/2448568/192360867-3c73d9b4-8114-4fdf-a2a3-e04bbc6c060c.png)

Reply:
Sorry, yes, I understand that. I just fail to see why it is an issue... isn't it okay if the constant ROUTE_NAME = "None" for unnamed routes? I guess I'm missing the point...

Reply:
We want this constant to represent the current route name as specified in our URL patterns. For unnamed routes, the string 'none' is not a falsy value in JavaScript and it would make more sense to have this value be an empty string.

Reply:
Thanks for explaining! Sorry it took me so long to understand. Updated.

Reply:
All good, the 'None' string arguably could have worked, but introduced a constraint to never have a route named 'None'. It's something we probably would not to, but an arbitrary constraint nonetheless.